includes/class-newspack-popups-inserter.php

```diff
@@ -38,8 +38,6 @@ public static function popup( $content = '' ) {
 		if ( $popup ) {
 			$markup  = self::generate_popup( $popup );
 			$content = self::insert_popup( $content, $popup );
-			wp_enqueue_script( 'amp-animation' );
-			wp_enqueue_script( 'amp-position-observer' );
```

Comment:
Without these scripts, I don't get any popups on non-AMP pages (e.g. using AMP transitional mode).

Reply:
Fixed in  https://github.com/Automattic/newspack-popups/pull/7/commits/fff33261233a95c4a3201371baf5a70c81891b05. If the AMP plugin isn't installed at all we will need  to define the scripts  as well,  but this has proven to be a little  tricky and  I'd like to address in a separate PR.